jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/ConnectionRetrySettings.scala

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.stream.connectors.jakartams
+
+import com.typesafe.config.Config
+import org.apache.pekko
+import org.apache.pekko.actor.{ ActorSystem, ClassicActorSystemProvider }
+import org.apache.pekko.util.JavaDurationConverters._
+
+import scala.concurrent.duration._
+
+/**
+ * When a connection to a broker cannot be established and errors out, or is timing out being established or
+ * started, the connection can be retried.
+ * All JMS publishers, consumers, and browsers are configured with connection retry settings.
+ */
+final class ConnectionRetrySettings private (
+    val connectTimeout: scala.concurrent.duration.FiniteDuration,
+    val initialRetry: scala.concurrent.duration.FiniteDuration,
+    val backoffFactor: Double,
+    val maxBackoff: scala.concurrent.duration.FiniteDuration,
+    val maxRetries: Int) {
+
+  /** Time allowed to establish and start a connection. */
+  def withConnectTimeout(value: scala.concurrent.duration.FiniteDuration): ConnectionRetrySettings =
```

Comment:
This API is intended for users to use, and usually user level parameters should be fully named instead of using `value`. The same goes for other places, such as `c` for config

Reply:
@laglangyue  Thanks for spotting this. I have updated please let me know if my understanding is right.

>  The same goes for other places, such as c for config

I'm not sure I follow. What's `c` for config are you referring to?